tests/integrations/mcs/scheduling/server_test.go

```diff
@@ -106,6 +106,7 @@ func (suite *serverTestSuite) TestAllocIDAfterLeaderChange() {
 	pd2, err := suite.cluster.Join(suite.ctx)
 	re.NoError(err)
 	err = pd2.Run()
+	re.NotEmpty(suite.cluster.WaitLeader())
```

Comment:
There are still some cases that are not judged empty, do we need also modify them or add comments?

Reply:
Added more waiting leaders for tests related to joining situation